test/e2e/generate_systemd_test.go

```diff
@@ -1,5 +1,3 @@
-// +build !remote
```

Comment:
Honestly, is there any point to enabling this one?

Reply:
`generate systemd` isn't supported for remote clients, so I think we can leave it fully disabled

Reply:
Reverted.